src/grt/src/fastroute/src/FastRoute.cpp

```diff
@@ -1167,6 +1168,11 @@ NetRouteMap FastRouteCore::run()
     }
 
     last_total_overflow = total_overflow_;
+
+    // generate DRC report each interval
+    if (drc_report_iter_step_ && i % drc_report_iter_step_ == 0) {
+      saveCongestion();
```

Comment:
Is this overwriting the same file each time?
(@oharboe is that what you want?)

Reply:
I modified to generate a file for each interval (eg. `congestion-step.rpt` where `step` is the iteration number)